packages/ember-glimmer/tests/integration/helpers/custom-helper-test.js

```diff
@@ -22,4 +27,333 @@ moduleFor('Helpers test: custom helpers', class extends RenderingTest {
     this.assertText('hello world');
   }
 
+  ['@test dashed helper can recompute a new value']() {
+    let destroyCount = 0;
+    let count = 0;
+    let helper;
+
+    this.registerHelper('hello-world', {
```

Comment:
This helper should probably take some input so that you can perform the I-N-U-R cycle described in #13127. 

```
this.register('hello-world', {
  ...
  compute([value]) {
   return `${value}-value`
  }
})
```

This way you can update it and reset it in the tests.

```
this.render(`{{hello-world foo}}`, {foo: 'hey'});

this.assertText('hey-value');

this.runTask(() => this.rerender());

this.runTask(() => set(this.context, 'foo', 'bye'));

this.assertText('bye-value');

this.runTask(() => set(this.context, 'foo', 'hey'));

this.assertText('hey-value');
```